apps/ledger-live-desktop/src/renderer/modals/AddAccounts/steps/StepChooseCurrency.jsx

```diff
@@ -63,7 +65,13 @@ const StepChooseCurrency = ({ currency, setCurrency }: StepProps) => {
 
   return (
     <>
-      {currency ? <CurrencyDownStatusAlert currencies={[currency]} /> : null}
+      {!navigator.onLine ? (
+        <div>
+          <ErrorBanner error={new NetworkDown()} />
```

Comment:
I don't know if it matters in React but my Angular spidey sense would put it outside the template, just before the return in the js part.

I would create a networkDownError const and use it there